cmd/loadTests.go

```diff
@@ -469,6 +497,7 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 
 			UserCreationTime := time.Since(startTime)
 			AverageUserCreationTime[threadIndex] += UserCreationTime
```

Comment:
I assume this is more `SumPerIndexUserCreationTime` than `AverageUserCreationTime`, do I get it right?

Reply:
@jhutar Yes, you are correct, the variable used to hold the average, now it is used to compute average, and really is a sum. I'll fix that. Thx